main/src/cgeo/geocaching/settings/Settings.java

```diff
@@ -1931,4 +1936,17 @@ public static int getListInitialLoadLimit() {
         return sensitiveKeys;
     }
 
+    public static void putLogTemplate(final String preferenceBaseName, final String title, final String text) {
+        putStringDirect(preferenceBaseName + ".Title", title);
+        putStringDirect(preferenceBaseName + ".Text", text);
+        putStringDirect(preferenceBaseName, title + ": " + text);
+    }
+
+    public static String[] getLogTemplate(final String preferenceBaseName) {
```

Comment:
Consider using Pair or ImmutablePair instead of an array

Reply:
Thanks, will look into that.